content/webapp/components/ItemRequestModal/RequestDialog.tsx

```diff
@@ -139,8 +149,9 @@ const RequestDialog: FunctionComponent<RequestDialogProps> = ({
               </p>
             </Space>
             <PickupDeadline>
-              Item requests need to be placed by 10am on the working day before
-              your visit. Please bear in mind the library is closed on Sundays.
+              <span data-testid="pickup-deadline">
+                {`${pickupDeadline}. Please bear in mind the library is closed on Sundays.`}
```

Comment:
small one, mostly to let you know; Since `pickUpDeadline` is always a string, you could just have

```suggestion
                {pickupDeadline}. Please bear in mind the library is closed on Sundays.
```